prepare_source

```diff
@@ -1,2 +1,6 @@
-apt_src python3.12
+# 3.12.7 isn't in testing yet. 
```

Comment:
is there a good reason why we need 3.12.7 now? no objection, just trying to understand the tradeoffs between fixed issues and effort of customized build

Reply:
It would take some days until the maintainer would push it into testing. The patch was accepted on the [3rd](https://tracker.debian.org/news/1571580/accepted-python312-3127-1-source-into-unstable/) of October and will probably land in testing on the 13th of October. I do not want to wait and get this out of the door now.